docs/en/index.md

```diff
@@ -39,7 +39,7 @@ We'll say that the following is true for the OLAP (online analytical processing)
 - Data is updated in fairly large batches (> 1000 rows), not by single rows; or it is not updated at all.
 - Data is added to the DB but is not modified.
 - For reads, quite a large number of rows are extracted from the DB, but only a small subset of columns.
-- Tables are "wide", meaning they contain a large number of columns.
+- Tables are "wide," meaning they contain a large number of columns.
```

Comment:
Bad merge.

Reply:
By typographic rules, commas should be inside the quotation marks. https://h.yandex-team.ru/?https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fstyle-guide%2Fpunctuation%2Fquotation-marks